Sprinkle of Yeezus/Settings.swift

```diff
@@ -11,52 +11,68 @@ import UIKit
 import UserNotifications
 
 class SettingsPage: UIViewController {
-    
 
-    @IBOutlet weak var InfoLabel: UILabel!
-    @IBOutlet weak var DescButton: UIButton!
-    @IBOutlet weak var DescLabel: UILabel!
-    @IBOutlet weak var Switch: UISwitch!
+    @IBOutlet private weak var InfoLabel: UILabel!
+    @IBOutlet private weak var DescButton: UIButton!
+    @IBOutlet private weak var DescLabel: UILabel!
+    @IBOutlet private weak var Switch: UISwitch!
+    @IBOutlet private weak var TimePicked: UIDatePicker!
     
-    var notificationsOn = Bool()
+    private var sprinkleTimePicked = Date()
+    private let notificationCenter = UNUserNotificationCenter.current()
```

Comment:
I noticed that we were using `UNUserNotificationCenter.current()` in multiple functions in this ViewController, so I added a `private let notificationCenter` so that it can be used in both spots. Makes the code cleaner and easier to read.